GLAA.Web/Controllers/PublicRegisterController.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using GLAA.Services.PublicRegister;
+using GLAA.ViewModels.PublicRegister;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace GLAA.Web.Controllers
+{
+    public class PublicRegisterController : Controller
+    {
+        private readonly IPublicRegisterViewModelBuilder publicRegisterViewModelBuilder;
+        private static PublicRegisterSearchViewModel currentSearchViewModel;
+
+        public PublicRegisterController(IPublicRegisterViewModelBuilder publicRegisterViewModelBuilder)
+        {
+            this.publicRegisterViewModelBuilder = publicRegisterViewModelBuilder;
+        }
+
+        [HttpPost]
+        public IActionResult Index(PublicRegisterLicenceListViewModel publicRegisterLicenceListViewModel, string submit)
```

Comment:
I think the var `submit` here is misleading. Is there a better name?

Reply:
changed to submitButtonType